test/e2e/garbage_collector.go

```diff
@@ -40,6 +42,38 @@ func getNonOrphanOptions() *v1.DeleteOptions {
 	return &v1.DeleteOptions{OrphanDependents: &falseVar}
 }
 
+func newOwnerDeployment(f *framework.Framework, deploymentName string) *v1beta1.Deployment {
+	zero := int64(0)
+	replicas := int32(2)
+	deploymentLabels := map[string]string{"app": "gc-test"}
+	return &v1beta1.Deployment{
+		ObjectMeta: v1.ObjectMeta{
+			Name: deploymentName,
+		},
+		Spec: v1beta1.DeploymentSpec{
+			Replicas: &replicas,
+			Selector: &metav1.LabelSelector{MatchLabels: deploymentLabels},
+			Strategy: v1beta1.DeploymentStrategy{
+				Type: v1beta1.RollingUpdateDeploymentStrategyType,
+			},
+			Template: v1.PodTemplateSpec{
```

Comment:
Looks like this function can share the same podTemplateSpec used by `newOwnerRC` to avoid duplicate code.